src/core/serializationHelpers.h

```diff
@@ -95,7 +96,23 @@ void load( Archive& ar, QString& s, const unsigned int )
 template< class Archive >
 void serialize( Archive& ar, QString& s, const unsigned int version )
 {
-    boost::serialization::split_free( ar, s, version );
+    split_free( ar, s, version );
+}
+
+template< class Archive >
+void serialize( Archive& ar, QUuid& uuid, const unsigned int /*version*/ )
+{
+    ar & make_nvp("data1", uuid.data1);
```

Comment:
It works as well, but I don't find it less verbose because then you need to split the serialization into 2 methods which takes as many lines of code in the end. This version might look a bit more low level but has the advantage of operating directly on the data and not doing successive type conversions QUuid -> QString -> string -> ...